jaxrs-api/pom.xml

```diff
@@ -24,14 +24,36 @@
     <artifactId>javax.ws.rs-api</artifactId>
     <version>2.2-SNAPSHOT</version>
     <packaging>${packaging.type}</packaging>
+
+    <parent>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0</version>
+    </parent>
+
     <name>javax.ws.rs-api</name>
+    <description>JAX-RS: Java API for RESTful Web Services (JAX-RS) is a Java programming language API spec that provides support in creating web services according to the Representational State Transfer (REST) architectural pattern.</description>
+    <inceptionYear>2007</inceptionYear>
 
     <url>https://github.com/eclipse-ee4j/jaxrs-api</url>
 
-    <organization>
-        <name>Eclipse Foundation</name>
-        <url>https://www.eclipse.org/org/foundation/</url>
-    </organization>
+    <developers>
+        <developer>
+           <id>mkarg</id>
+           <name>Markus KARG</name>
```

Comment:
This person is active in the project and should be given credits for his work.

Reply:
@lukasj That section of the POM is not about credits for our work. As the official POM reference said, this is a list of people that serve as contacts of the project. I did not add me for credits, but simply as an anchor point. That's why I am good with getting replaced by a generic name instead of adding a list of lots of people.

Reply:
What the reference says and what projects really use this for may be two different things. I, personally, am looking for contacts in mailing lists section primarily and checking developers section just for pointers to blogs etc. If I turn it the other way around - shouldn't project leads (who are listed on https://projects.eclipse.org/projects/ee4j.jaxrs/who anyway) be those to contact 'by default'?

Reply:
I will not discuss using POM items differently than officially described.

Regarding projects leads: Not necessarily. Once the project leads agreed to daily serve after being democratically elected, yes. If they carry their burden only due to historic reasons and possibly do not really want to actually be the daily target of lots of new contributors asking silly beginners questions, then no.

Reply:
This is not about interpreation or discussion. The role of the project lead is described here https://www.eclipse.org/projects/dev_process/#4_6_2_PL:
>Eclipse projects are managed by one or more project leads. Project leads are responsible for ensuring that their project's committers are following the Eclipse Development Process, and that the project is engaging in the right sorts of activities to develop vibrant communities of users, adopters, and contributors.

In the Eclipse Foundation, project leads are project managers, not *necessarily* developers, not even *necessarily* committers or contributors.

So just because the EF asks someone to *manage* a project does not automatically make him responsible for *code*, neither he automatically assumes he feels responsible to answer questions about the *code*. Certainly this *typically will* be the case, but that's not the point.

That's why I originally proposed that people *willing* to be contacted about the code simply shall send a PR so their will to be contacted is clearly documented, and that's why I explicitly mentioned @chkal and @spericas . Neither of them send a PR so far. I cannot simply put @chkal and @spericas in that list without their upfront consent.